### ISSUE TITLE
GRPO lora

### ISSUE THREAD

**Issue by Yetaiheng** [modelscope/ms-swift#4593]
我的命令行
{   // Qwen2.5-VL-3B-Instruct
  "version": "0.2.0",
  "configurations": [
    {
      "name": "Python: Swift SFT GRPO Debug",
      "type": "python",
      "request": "launch",
      "program":"/opt/nas/n/local/yetaiheng/ms-swift-main/ms-swift-main/swift/cli/rlhf.py",
      "args": [
        "--rlhf_type", "grpo",
        "--model", "/opt/nas/n/local/yetaiheng/ms-swift-main/ms-swift-main/Qwen2.5-VL-3B-Instruct",
        "--external_plugins", "/opt/nas/n/local/yetaiheng/ms-swift-main/ms-swift-main/examples/train/grpo/plugin/plugin.py",
        "--reward_funcs", "format", // external_r1v_acc
        "--train_type", "lora",
        "--torch_dtype", "bfloat16",
        "--dataset","okwinds/clevr_cogen_a_train",
        "--max_completion_length", "1024",
        "--num_train_epochs", "1",
        "--per_device_train_batch_size", "4",
        "--per_device_eval_batch_size", "4",
        "--learning_rate", "1e-6",
        "--gradient_accumulation_steps", "2",
        "--save_strategy", "steps",
        "--eval_strategy", "steps",
        "--eval_steps", "1000",
        "--save_steps", "1000",
        "--save_total_limit", "10",
        "--logging_steps", "1",
        "--output_dir", "outputs/GRPO_CLEVR_COUNTDOWN",
        "--warmup_ratio", "0.01",
        "--dataloader_num_workers", "4",
        "--num_generations", "4",
        "--temperature", "1.0",
        "--system", "examples/train/grpo/prompt.txt",
        "--deepspeed", "zero2",
        "--log_completions", "true",
        "--num_iterations", "1",
        "--async_generate", "false",
        "--beta", "0.001"
      ],
      "python": "/opt/nas/p/conda/envs/yth-swift/bin/python",
      "env": {
        "CUDA_VISIBLE_DEVICES": "0",  // 指定GPU
        "PYTHONPATH": "/opt/nas/n/local/yetaiheng/ms-swift-main/ms-swift-main"  // 确保项目路径包含Swift源码
      },
      "justMyCode": false  // 允许调试第三方库代码
    }
  ]
}

报错：
/opt/nas/p/conda/envs/yth-swift/lib/python3.10/site-packages/torch/utils/checkpoint.py:86: UserWarning: None of the inputs have requires_grad=True. Gradients will be None

能运行但是感觉不对，训练中间输出：
{'loss': 1.4e-06, 'grad_norm': 1.13720572, 'learning_rate': 2e-08, 'memory(GiB)': 10.64, 'train_speed(iter/s)': 0.089636, 'completions/mean_length': 114.0, 'completions/min_length': 87.0, 'completions/max_length': 200.0, 'completions/clipped_ratio': 0.0, 'rewards/Format/mean': 0.625, 'rewards/Format/std': 0.51754922, 'reward': 0.625, 'reward_std': 0.53867513, 'kl': 0.00123596, 'clip_ratio/low_mean': 0.0, 'clip_ratio/low_min': 0.0, 'clip_ratio/high_mean': 0.0, 'clip_ratio/high_max': 0.0, 'clip_ratio/region_mean': 0.0, 'epoch': 0.0, 'global_step/max_steps': '8/34650', 'percentage': '0.02%', 'elapsed_time': '1m 27s', 'remaining_time': '4d 8h 48m 58s'}

求助有没有大佬解答！！！

**Comment by hjh0119** [comment]
This warning is related to gradient checkpointing and can be ignored.
